### PR TITLE
libtatsu 1.0.4 (new formula)

### PR DESCRIPTION
**Pull request by tux-mind**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/lib/libtatsu.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  sha256 "08094e58364858360e1743648581d9bad055ba3b06e398c660e481ebe0ae20b3"
+  license "LGPL-2.1-or-later"
+  head "https://github.com/libimobiledevice/libtatsu.git", branch: "master"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libplist"
+
```
Comment (chenrui333) [inline Formula/lib/libtatsu.rb:14]:
```suggestion
  depends_on "libplist"

  uses_from_macos "curl"
```

Comment (tux-mind) [reply Formula/lib/libtatsu.rb:14]: Thank you <3

**Comment by tux-mind** [comment]
Is it possible to remove the notable repository requirement from the Formulae audit?
```
* GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
```

The `libtatsu` repo is from libimobiledevice organization, whose main repo has 1.3k forks, 251 watchers and 6.7k stars ⭐ 

Thank you in advance for your precious help!

**Comment by jpstotz** [comment]
> The `libtatsu` repo is from libimobiledevice organization, whose main repo has 1.3k forks, 251 watchers and 6.7k stars ⭐

IMHO this is a bug of homebrew as it does not consider dependencies. `libtatsu` is a mandatory requirement of [libimobiledevice](https://github.com/libimobiledevice/libimobiledevice) which has 6.7k stars and 1.3k forks. Thus if a popular project requires a dependency the popularity of the dependency should be the cumulated popularity of all projects that depend on it, not just the project itself.

**Comment by tux-mind** [comment]
Thanks @daeho-ro for adding the CI label, all the checks pass!

@chenrui333 can you please review and approve this PR?

Thank you for your help 😊

**Review by MikeMcQuaid — APPROVED** [review]
Thanks for explaining the reasoning here! Looks good to me.

**Pull request merged**